pkgs/development/python-modules/wikitextparser/default.nix

```diff
@@ -0,0 +1,42 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pytestCheckHook
+, regex
+, wcwidth
+}:
+
+buildPythonPackage rec {
+  pname = "wikitextparser";
+  version = "0.54.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "5j9";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-AGQfjUNxeleuTS200QMdZS8CSD2t4ah5NMm9TIYjVHk=";
+  };
+
+  propagatedBuildInputs = [
+    wcwidth
+    regex
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "wikitextparser" ];
+
+  meta = {
+    homepage = "https://github.com/5j9/wikitextparser";
+    description = ''
```

Comment:
I think this sentence is a bit long for description.

How about citing "A simple parsing tool for MediaWiki's wikitext markup"?
https://github.com/5j9/wikitextparser/blob/9052785262daa333404e302c5f601729d45525ac/pyproject.toml#L12

Reply:
Sounds good.